packages/gatsby/src/commands/develop.ts

```diff
@@ -349,7 +353,17 @@ async function startServer(program: IProgram): Promise<IServer> {
   return { compiler, listener, webpackActivity }
 }
 
+const feedbackHandler = async (exitCode: number | string): Promise<void> => {
+  if (await userPassesFeedbackRequestHeuristic()) {
+    showFeedbackRequest()
+  }
+  process.exit(exitCode === 1 ? 1 : 0)
```

Comment:
this is confusing - if exitCode won't be neither 0 nor 1, we will exit with 0 (which is wrong?). Also why do we need to call `process.exit` here at all?

Reply:
exitCode can be a string when coming from a signal responder (e.g. SIG). So i'm basically just saying, if the exitCode was 1, exit with 1, else exit with 0. But seems like i have some refactoring to do here either way.